CHANGELOG.md

```diff
@@ -7,21 +7,31 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [0.9.0] - 2021-08-10
 ### Added
+- Add topolvm helm charts (#302)
 - Add recommended labels (#320)
 - Add Storage Capacity Tracking mode (#315)
 
 ### Changed
+- support k8s 1.21 (#299)
+- update conformed csi version (#322)
 - Add error message (#314)
-- add document to run test using minikube for lvmd daemonset (#318)
 - Remove CSI Attacher sidecar (#319)
-  - **BREAKING**: You need to recreate the CSIDriver resource because of CSI Attacher sidecar was removed from topolvm.
+  - **BREAKING**: You need to recreate the CSIDriver resource because CSI Attacher sidecar was removed from topolvm.
+
+### Removed
+- Remove kustomize manifests. (#336)
 
 ### Fixed
+- add document to run test using minikube for lvmd daemonset (#318)
+- Fix lint error (#321)
+- Migrate E2E manifests from kustomize to Helm (#325)
```

Comment:
Shouldn't this issue be categorized as `Changed`? Shouldn't we announce changes of deployment procedure?